orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/ExplicitRollback.groovy

```diff
@@ -54,9 +55,24 @@ class ExplicitRollback implements Rollback {
   ApplySourceServerGroupCapacityStage applySourceServerGroupCapacityStage
 
   @JsonIgnore
-  def List<Stage> buildStages(Stage parentStage) {
-    def stages = []
+  List<Stage> buildStages(Stage parentStage) {
```

Comment:
Is this method part of an interface?

Reply:
It is ... not sure why it didn't have an `@Override`